Assignments/SCF-Session-2/FCFS/JobScheduler.java

```diff
@@ -0,0 +1,175 @@
+package FCFS;
+
+import java.util.Scanner;
+
+class FirstComeFirstServe
+{
+	
+	int[] findWaitingTime (int processes, int inputTimes[][])
+	{
+		int[] serviceTime, waitingTime;
+		
+		serviceTime = new int[processes];
+		serviceTime[0] = 0;
+		waitingTime = new int[processes];
+		waitingTime[0] = 0; //waiting time for first process is 0
+		
+		for(int i = 1; i < processes; i++)
+		{
+			if(inputTimes[i - 1][0] <= serviceTime[i - 1])
```

Comment:
Hard-coded values should not be used like `1` in `serviceTime[i - 1]`.

Reply:
done